tests/test_dynamics_client.py

```diff
@@ -20,7 +23,7 @@
     PermissionDenied,
     WebAPIUnavailable,
 )
-from dynamics.test import MockClient, ResponseMock
+from dynamics.test import MockClient, ResponseMock, environ
```

Comment:
`environ` does not need to be imported, as pytest dependency injection system will inject it to the test if environ is requested as a parameter to the test function.